graylog2-web-interface/src/components/indices/IndexSetFieldTypes/originExpandedSections/OverriddenIndexExpandedSection.tsx

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (C) 2020 Graylog, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the Server Side Public License, version 1,
+ * as published by MongoDB, Inc.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * Server Side Public License for more details.
+ *
+ * You should have received a copy of the Server Side Public License
+ * along with this program. If not, see
+ * <http://www.mongodb.com/licensing/server-side-public-license>.
+ */
+import React from 'react';
+
+import type { ExpandedSectionProps } from 'components/indices/IndexSetFieldTypes/types';
+
+const OverriddenProfileExpandedSection = ({ type, fieldName }: ExpandedSectionProps) => (
+  <span>
+    Field type mapping <b>{fieldName}: </b><i>{type}</i> comes from the individual, custom field type mapping.
```

Comment:
I think we can just highlight the field type here instead of repeating the field name, it would look a but cleaner:

Before:
![image](https://github.com/Graylog2/graylog2-server/assets/46300478/f7856cdb-8a8b-4cc3-bfd5-c1648a13b2ea)

After:
![image](https://github.com/Graylog2/graylog2-server/assets/46300478/c9775f70-82f0-4710-a876-2e855547b36d)

Reply:
@linuspahl Then we would have to remove the word `mapping` as well,  just `Field type IP`